sudokuProject/src/sudoku/Settings.java

```diff
@@ -1,5 +1,69 @@
 package sudoku;
 
+import org.w3c.dom.DOMException;
+
+import sudoku.io.XMLHandler;
+
 public class Settings {
+	private String		level;
+	private String		output;
+	private String		algorithm;
+	private XMLHandler	xml;
 
+	public Settings(String level, String algorithm, String output) throws Exception
+	{
+		this.level = level;
+		this.output = output;
+		this.algorithm = algorithm;
+		xml = new XMLHandler(level, algorithm, output);
+	}
+	public Settings() throws Exception
+	{
+		this.level = "1";
```

Comment:
What is the meaning of this "1" remember that hardcoded values does not help the readability